### ISSUE TITLE
Althor's Abacus Vanished from Equipped Items (Character Screen)

### ISSUE THREAD

**Issue by SkriemNL** [SunwellTracker/issues#3405]
Frosthold - Alliance 
![Abacus](https://user-images.githubusercontent.com/73081800/96443088-5b8ff280-120c-11eb-8e38-d8e5e9f6389f.PNG)

Hi Devs,

The Althor's Abacus (264) that I won the 8th of October Vanished from my character screen 1 week later while we were doing Gunship heroic. 

Attached picture as proof i had it. 

To fully explain what happened: 
This happened during our Guildrun (Ascension, Alliance) 
We were doing Gunship heroic in ICC25 on the 15th of October.

I had just upgraded my Ashen Verdict Ring to 277 after getting exalted.
Went to the gunship encounter, equipped, enchanted and gemmed the Ring.
I checked in my character screen my rings and Spellpower gain and saw Abacus still equipped.
(looking at rings you would see an empty slot right underneath the rings (slot 13) 

Closed my character screen and we switched for Gunship to Heroic, started the event and wiped.
Around 20:45 to 20:55 we did our second pull on Heroic Gunship and we didn’t see the health frame of our ship (we did however see the health frame of the Horde Ship). 
We again wiped but the encounter got stuck, No one was able to enter the instance with the notice: “Encounter in progress” 

We tried all sorts of thing:
switching between normal and heroic multiple times 
switching to 10man and entering and then out and back to 25
Made sure everyone had released
Waited for 5-10 minutes
remade the entire Raid 

None of it worked and the encounter was still stuck.
I then made a ticket in game and Crafty came to help us.
(Didn’t know how long it would take to get help, so some were flying to TOC already, I never reached TOC or any vendor) 

Crafty Teleported us back outside of the instance and asked us to wait for 5 minutes so he could investigate. He unlocked the encounter somehow and we were able to enter again.

As we were clearly not ready for heroic and didn’t want to bug the encounter again, we killed gunship normal. 

Then on Saurfang right before pull I checked my character screen again and saw an empty trinket slot. My Althor's Abacus was suddenly gone. Checked bags – Nothing.

When Crafty came to help us one of our members started making jokes to Crafty that he should give us an Abacus. So at first we thought Crafty pulled a Joke.. 

I then made another ticket requesting for the return of my trinket.
As Crafty thought it was a joke it wasn’t looked at immediately and only after asking him on Discord the next day he understood that I was not joking. 

(I understand now that it wasn’t him taking the trinket) 

He tried to help me to find it back and tried the following things: 
Checked mail, bags and bank.
Disabled all addons -> checked all bags and bank again
Typed /equip Althor's Abacus with bags and bank open.
Checked deleted items webpage
But its nowhere to be found.

What I understood from Crafty is that if it doesn’t show as a deleted item it can only be disenchanted or sold to a vendor, in both cases not traceable. 

Had it equipped before the encounter 100% sure 
I never unequip an item so an empty slot in my character screen doesn’t happen ever. 
(As in by manually dragging items to my bag.) 

The only way I switch gear is with the build in equipment manager. (no addons) 
I did not use the equip manager at that time as the upgraded ring still isn’t saved in my resto set.
And did not switch any gear sets during this time. 

I did not go to any vendor during this time so selling it is impossible.
(no vendor at or near Gunship to my knowledge and after it got stuck I was just outside of ICC) 
And not using any addons that sell gear either, (only ElvUI Autorepair).

Also did not disenchant any gear during this time.
I have my Disenchant keybound to SHIFT+0 (Zero) 
Which would take my left hand off WASD and other keybinds. 
Not something you accidentally press during a boss encounter (with your character screen open?)

I rarely Disenchant during raids as I only leveled enchanting for the ring enchants and not the master looter or the one that disenchants gear during raids.

As I did not Vendor anything during the timespan -> 100% sure
Did not switch any trinkets during while in ICC -> 100% sure
Was not using the equipment manager at this time -> 100% sure
Never Deleted any gear accidentally -> 100% sure
And did not disenchant anything during this time -> 100% sure

Exactly the encounter that can drop Abacus bugged and was reset, and the tinket is gone.

I think something strange happened while we were switching normal to heroic and back multiple times. People not able to enter because they had not done LK normal yet (including me) Bugging the encounter and getting stuck. Or while Crafty was rolling back the encounter.

Then a little side story: 
In all the guild runs of TOC25 (normal and heroic) over the past many months, Not once did solace drop, because of this the trinkets of all our healers are lacking behind. 

Ive calculated this and with a dropchance of 4% statistically we should have seen atleast 2-3 Solace drops considering the amount of TOC runs we did with mains and alts. always had main healers in every run to farm it. 

So we have been very unlucky to say the least. and now Abacus dropping (first healer trinket dropping since the launch of TOC) and then losing it in such a way isnt just a big hit for me but for all the healers in guild (and Guild).

I for example was still healing with a naxx trinket as all the Ulduar 25 trinkets were more beneficial for other healers.

After Crafty tried to help me he suggested me to create a ticket here as he is not allowed to help me to restore it as he can’t see anything wrong with the database.

In short, we are hoping for the return of Abacus
(Even if it costs costs me the 3 free item restore points or the 500 frosthold coins that i still have) 

Thanks for reading.

Skriem 
Officer of Ascension
Frosthold, Alliance

**Comment by SkriemNL** [comment]
Any news ?

**Comment by SkriemNL** [comment]
>_<

**Comment by Azathothh** [comment]
I'm pretty sure that Crafty already explained everything.
We've checked everything that we could and the only logical conclusion for us is that it was either disenchanted or sold. It could be some missclick, or addon malfunction or using /script command from some troll ,we can't check that as we're not logging every single action (that would require enormous storage space). I've been gm for many years, done tens of thousands of tickets and never ever seen an item disappearing just like that. It was always either sold/deleted/disenchanted/sent to mailbox if it's being unequipped, but there are no empty slots in bags.
I'm really sorry to disappoint you but we can't help with this case...
